.github/workflows/ci.yaml

```
@@ -96,9 +96,9 @@ jobs:
         id: tag
         run: |
           if echo "${GITHUB_REF}" | grep ^refs/heads/main$ >/dev/null; then
-            echo ::set-output name=tag::latest
+            echo "tag=latest" >> $GITHUB_OUTPUT
           elif echo "${GITHUB_REF}" | grep ^refs/tags/v >/dev/null; then
-            echo ::set-output name=tag::"$(echo "${GITHUB_REF}" | sed "s/refs\/tags\/v//")"
+            echo "tag=$(echo \"${GITHUB_REF}\" | sed \"s/refs\/tags\/v//\")" >> $GITHUB_OUTPUT
```

```suggestion
            echo "tag=${GITHUB_REF}" | sed "s|refs/tags/v||" >> $GITHUB_OUTPUT
```

Maybe this could help simplify it somewhat?

---

done